cmd/pulumi-test-language/providers/parameterized_provider.go

```diff
@@ -92,6 +100,9 @@ func (p *ParameterizedProvider) GetSchema(
 				},
 			},
 		},
+		Language: map[string]schema.RawMessage{
+			"go": goPkgInfo,
+		},
```

Comment:
Sorry, but hard rule for conformance tests is _no_ language specific flags. This needs to work _without_ anything being put into the Languages block here.

Reply:
I'm not sure how we can do that because it is specific to how the paramaterization changes the behaviour for this specific language :\